source/Ship.cpp

```diff
@@ -239,13 +239,20 @@ void Ship::Load(const DataNode &node)
 				if(child.Size() >= 2)
 					outfit = GameData::Outfits().Get(child.Token(1));
 			}
-			Angle gunPortAngle = Angle(0.);
+			vector<Angle> angles{};
```

Comment:
I prefer 3 separate variables or 2 separate variables where the turnRange is a pair.

turnRange.first is more readable than angle[1] in the code below.